R/reporter-zzz.R

```diff
@@ -59,7 +59,7 @@ with_reporter <- function(reporter, code, start_end_reporter = TRUE) {
 
 stop_reporter <- function(message) {
   signal(message, "testthat_abort_reporter")
```

Comment:
Is signaling a `message` with cli inline markup problematic at all?

Reply:
I don't think anything actually uses the message, so might be safer to set to `""`.

Reply:
@lionel- could `signal()` allow `message` to be missing, i.e. `signal(class = "testthat_abort_reporter")`? I also just had a case in dplyr where I just wanted to do the signaling but didn't care about the message

Reply:
I think it'd be fine to take `""` by default.

Reply:
Actually, I can just combine the `cli_abort()` and `signal()` — I think this must be a remnant from when the code used `stop()` and it was harder to class the error.